netsync/manager.go

```diff
@@ -191,9 +199,26 @@ func (sm *SyncManager) startSync() {
 
 		syncPeer.PushGetBlockLocatorMsg(syncPeer.SelectedTip(), sm.dagParams.GenesisHash)
 		sm.syncPeer = syncPeer
-	} else {
-		log.Warnf("No sync peer candidates available")
+		return
+	}
+
+	log.Warnf("No sync peer candidates available")
+	if sm.dag.AdjustedTime().Sub(sm.dag.CalcPastMedianTime()) > time.Minute {
```

Comment:
Extract this calculation to some function, the sending of selectedTip to another function and time.Minute to a constant.